lib/walker.ts

```diff
@@ -194,7 +195,22 @@ function upon(p: string, base: string) {
 }
 
 function collect(ps: string[]) {
-  return globSync(ps, { absolute: true, dot: true });
+  const patterns: string[] = [];
+  const paths: string[] = [];
+
+  for (const pattern of ps) {
+    if (isDynamicPattern(pattern)) {
+      patterns.push(pattern);
+    } else if (existsSync(pattern)) {
+      paths.push(pattern);
+    }
+  }
+
+  if (patterns.length !== 0) {
+    return [...paths, ...globSync(patterns, { absolute: true, dot: true })];
+  }
+
+  return paths;
```

Comment:
I'm wondering why tinyglobby cannot use `isDynamicPattern` his own by default before starting parsing a pattern?

Reply:
consistency shenanigans, but mainly just that i don't think it will be needed once its optimization pr gets fixed and lands